### PR TITLE
379 controls styling using resources dictionary

### PR DESCRIPTION
**Pull request by Sofiya-kumar**
- Added support for control styling
- Added config file to pass the custom styling

### CODE REVIEW

File: Trimble.Modus.Components/Hosting/AppConfig.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Trimble.Modus.Components.Hosting;
+
+public class AppConfig
```
Comment (rthanga1) [inline Trimble.Modus.Components/Hosting/AppConfig.cs:3]: Can we have this named `ModusConfig`? Or something like that?

File: DemoApp/DemoApp/Views/CheckBoxPage.xaml
Comment (rthanga1) [inline DemoApp/DemoApp/Views/CheckBoxPage.xaml:1]: So many files are deleted. Is this intentional?

Comment (Sofiya-kumar) [reply DemoApp/DemoApp/Views/CheckBoxPage.xaml:1]: Those files were duplicated.

File: Trimble.Modus.Components/Hosting/AppConfig.cs (hunk 1 above)
Comment (Sofiya-kumar) [reply Trimble.Modus.Components/Hosting/AppConfig.cs]: Modified to ModusConfig

**Review by rthanga1 — APPROVED** [review]

**Pull request merged**